### DISCUSSION TITLE
I have some idea to fix problem when tag bookmark sliding accidentally

### DISCUSSION THREAD

**Discussion by HypeNeutron** [alefragnani/vscode-bookmarks#406] [Ideas]
Idea that is  can moveable bookmark, What do you think?

**Comment by alefragnani** [comment]
Hi @RedNeutorn ,

I’m not sure I understood your request. What do you mean about _sliding accidentaly_?

Are the bookmarks not respecting inserts/deletes above them, and not moving down/up as they should, or you accidentally bookmarked one line, but in fact would like that bookmark to be a few lines below/above and need to _move_ the bookmark down/up manually?
